### PR TITLE
Fixes #17:Make the site mobile friendly

### LINKED ISSUE #17: Make the site mobile friendly
The site shows inconveniently large font on mobile devices and the navbar minimized side menu isn't working too.

### PR DESCRIPTION
**Pull request by arc9693**

### CODE REVIEW

File: css/styles.scss [hunk 1]
```diff
@@ -46,20 +46,21 @@ html, body {
 }
 
 .page-content {
-  width: 70%;
+//  width: 80%;
```
Comment (pandusonu2) [inline css/styles.scss]: Never commit commented code

**Review by pandusonu2 — COMMENTED** [review]
I guess @nkprince007 will give a better review for this

**Comment by naveenkumarsangi** [comment]
@arc9693 could you please rebase your pull request on latest master?

File: css/styles.scss [hunk 2]
```diff
@@ -107,3 +107,31 @@ html, body {
     }
   }
 }
+
+.navbar-fixed{
```
Comment (naveenkumarsangi) [inline css/styles.scss]: indentation before the bracket is a little off, please maintain consistency with existing code.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+(function($) {
+  $(function() {
+
+    $('.my-btn').sideNav({
```
Comment (naveenkumarsangi) [inline js/main.js]: `my-btn` is a weird name. Please use something more consistent and meaningful, like a `menu-btn` or something.

File: css/styles.scss (hunk 2 above)
Comment (arc9693) [reply css/styles.scss]: sure.

Comment (naveenkumarsangi) [reply css/styles.scss]: This isn't fixed yet and no separate commits to fix this, please. Do that in the commit that actually introduces this change.

**Review by naveenkumarsangi — APPROVED** [review]
No separate commits for fixing indentation, please.

**Pull request merged**